src/examples/index.js

```diff
@@ -1,6 +1,5 @@
 /* global window:false */
 
-import 'babel-polyfill'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
babel/preset-env is set to add polyfills to entry files

Reply:
It reads differently in [the documentation](https://babeljs.io/docs/en/babel-preset-env#usebuiltins-entry)

> This option enables a new plugin that replaces the statement `import "@babel/polyfill"` or `require("@babel/polyfill")` with individual requires for `@babel/polyfill` based on environment.

Reply:
You're right. I read it wrong

Reply:
Fixed